src/crewai/__init__.py

```diff
@@ -18,6 +22,46 @@
     category=UserWarning,
     module="pydantic.main",
 )
+
+_telemetry_submitted = False
+
+def _is_tracking_disabled():
+    """Check if tracking should be disabled based on environment variables."""
+    return (
+        os.getenv("CREWAI_DISABLE_TELEMETRY", "false").lower() == "true"
+        or os.getenv("CREWAI_DISABLE_TRACKING", "false").lower() == "true"
+    )
+
+
+def _track_install():
+    """Track package installation/first-use via Scarf analytics."""
+    global _telemetry_submitted
+    
+    if _telemetry_submitted or _is_tracking_disabled():
+        return
+    
+    try:
+        pixel_url = "https://api.scarf.sh/v2/packages/CrewAI/crewai/docs/00f2dad1-8334-4a39-934e-003b2e1146db"
+        
+        req = urllib.request.Request(pixel_url)
+        req.add_header('User-Agent', f'CrewAI-Python/{__version__}')
+        
+        with urllib.request.urlopen(req, timeout=2):  # nosec B310
+            _telemetry_submitted = True
+            
+    except (URLError, OSError, Exception):
```

Comment:
I think you can keep `Exception` onlye
```suggestion
except Exception:
```